extension/js/common/platform/xss.ts

```diff
@@ -230,6 +230,15 @@ export class Xss {
     return str.replace(/&/g, '&amp;').replace(/"/g, '&quot;').replace(/'/g, '&#39;').replace(/</g, '&lt;').replace(/>/g, '&gt;').replace(/\//g, '&#x2F;');
   };
 
+  public static stripEmojis = (str: string) => {
+    const emojiRegex = /[\u{1F300}-\u{1F64F}\u{2700}-\u{27B0}\u{23E9}-\u{23FA}\u{2194}-\u{21AA}\u261D\u26A1\u263A-\uD83E\uD83E]/gu;
```

Comment:
It'll be better to use https://github.com/mathiasbynens/emoji-regex for emoji regex, as periodically some new emojis are added and our static regex won't include them, while this lib is regularly updated to include all existing emojis.
For example, this regex doesn't detect 🥴 emoji

Reply:
Thank you for the remarks. Yes, I actually have problems with the ranges of emojis to remove - I'll check the repo you've shared and implement it here.

Reply:
@sosnovsky After updating the `tsconfig.json` and including this line `"emoji-regex": ["../node_modules/emoji-regex/index.d.ts", "../node_modules/emoji-regex/index.js", "COMMENT"]` in the compilerOption -> paths, I'm still getting the following error: 
> Cannot find module 'emoji-regex'. Did you mean to set the 'moduleResolution' option to 'nodenext', or to add aliases to the 'paths' option?

I have checked `node_modules/emoji-regex` and those files are present. I'm sorry, may I hear your suggestions on how I could proceed further?

Reply:
I think I have found it out. Thanks!